memor/params.py

```diff
@@ -143,15 +143,28 @@ class LLMModel(Enum):
     CODE_LLAMA_70B_PYTHON = "code-llama-70b-python"
 
     MISTRAL_7B = "mistral-7b"
+    MISTRAL_7B_INSTRUCT = "mistral-7b-instruct"
     MIXTRAL_8X7B = "mixtral-8x7b"
+    MIXTRAL_8X7B_INSTRUCT = "mixtral-8x7b-instruct"
     MIXTRAL_8X22B = "mixtral-8x22b"
+    MIXTRAL_8X22B_INSTRUCT = "mixtral-8x22b-instruct"
     MISTRAL_NEMO = "mistral-nemo"
-    MISTRAL_TINY = "mistral-tiny"
-    MISTRAL_SMALL = "mistral-small"
-    MISTRAL_MEDIUM = "mistral-medium"
-    MISTRAL_LARGE = "mistral-large"
-    CODESTRAL = "codestral"
-    PIXTRAL = "pixtral-12b"
+    MISTRAL_NEMO_INSTRUCT = "mistral-nemo-instruct"
+    MISTRAL_SMALL_24B = "mistral-small-24b"
+    MISTRAL_SMALL_24B_INSTRUCT = "mistral-small-24b-instruct"
+    MISTRAL_SMALL_3_1_24B = "mistral-small-3.1-24b"
+    MISTRAL_SMALL_3_1_24B_INSTRUCT = "mistral-small-3.1-24b-instruct"
```

Comment:
Add `Mistral-Small-Instrcut`.
https://huggingface.co/mistralai/Mistral-Small-Instruct-2409

Reply:
ok, added in 36f2add.